index.js

```diff
@@ -27,9 +27,11 @@ if (IS_SA_TOKEN && !(DOPPLER_PROJECT && DOPPLER_CONFIG)) {
 
 const secrets = await fetch(DOPPLER_TOKEN, DOPPLER_PROJECT, DOPPLER_CONFIG);
 
-for (const [key, value] of Object.entries(secrets)) {
+for (const [key, secret] of Object.entries(secrets)) {
+  const value = secret.computed || secret.raw || "";
```

Comment:
Was there a specific situation you've run into where falling back to the `raw` secret value would be desired? That's pretty much only useful if you want to see the raw pre-computed secret reference (e.g., `0.0.0.0:${PORT}` rather than `0.0.0.0:8080`). In this context, I don't think you'd ever really want that. There might be situations where `raw` is empty and `computed` is not, but not the reverse.

Reply:
Ah, yes this was probably from my debugging. I'll remove that part.

Reply:
@watsonian Updated